### ISSUE TITLE
auto kernel tuner integration:IR support

### ISSUE THREAD

**Issue by AlisaChen98** [microsoft/nnfusion#117]
ETA
   * IR support 
     - [x] Dot 
     - [x] BatchNormInference 
     - [x] Sum 
     - [x] Pad 
     - [x] AvgPool
     - [x] Softmax
     - [x] BatchMatMul
     - [x] Convert
     - [x] GatherV2
     - [x] OneHot
     - [x] Elementwise: Divide,Multiply, Sigmoid, Tanh,Power,Subtract,Rsqrt,Divide,Relu6
     - [x] MaxPool

**Comment by nnfbot** [comment]
Thanks for the report @AlisaChen98! I will look into it ASAP! (I'm a bot).
